lib/force.go

```diff
@@ -634,6 +634,16 @@ func (f *Force) GetSobject(name string) (sobject ForceSobject, err error) {
 	return
 }
 
+func (f *Force) RestQuery(url string) (result string) {
```

Comment:
This is almost the same as the `GetREST` function I just added.  Want to remove one of them?

Reply:
Yikes!  Great minds and all that. Let me take a look.

Reply:
Might be why I had so many test problems.

Reply:
Let me test this out. I have implemented a function that will leverage the SFDX CLI project environment authentication.  This revealed an issue with the instance url due to differences between what is stored for SFDX CLI and Force CLI.  

Wow! They are almost letter for letter the same.

The only thing they both suffer from, and maybe using path.Join solves this is leading back slashes on the url passed in.

I'm happy to remove mine.  

Thanks @cwarden

Reply:
Oh, by the way, how do you invoke your comnand?

Reply:
`GetREST` can be called from the command line using `force get /some/rest/endpoint`.  I also added `PostREST` and `force post` that reads json to post from stdin.